include/spirv-tools/libspirv.h

```diff
@@ -374,6 +374,8 @@ typedef enum {
   SPV_ENV_UNIVERSAL_1_0,  // SPIR-V 1.0 latest revision, no other restrictions.
   SPV_ENV_VULKAN_1_0,     // Vulkan 1.0 latest revision.
   SPV_ENV_UNIVERSAL_1_1,  // SPIR-V 1.1 latest revision, no other restrictions.
+  SPV_ENV_OPENCL_1_2,     // OpenCL 1.2 plus cl_khr_il_program, latest revision.
```

Comment:
Lets avoid reordering values of the other enums in the list, so please put the new ones at the end.

Reply:
Please put a comment at the top of the enum declaration saying that by default we mean the OpenCL full profile.

Reply:
I edited the comment next to each OPENCL flag to indicate it means the full profile: is that enough do you prefer having a comment at the top as well?